### DISCUSSION TITLE
유니티에서 제페토 아바타 불러오기 할 때 스크립트 작성에서 막히네요ㅠ

### DISCUSSION THREAD

**Discussion by phy9051** [naverz/zepeto-studio-kor#646] [5. Q&A]
안녕하세요. 제페토 스튜디오에서 제공하는 내용 따라서
유니티에 제페토 아바타 불러오기 해보고 있는데
스크립트 작성 단계에서 막히네요ㅠ

혹시 아래 이미지와 같이,
제트브레인으로 스크립트 작성할 때
망치 아이콘과 함께 생성되는 'Generate argument stubs' 옵션이 안 보일 때는
해당 옵션을 어떻게 생성할 수 있는지 아시는 분 있나요?

코딩 잘 몰라도 이 부분만 찾아서 해결할 수 있는지
응답 부탁드려요ㅠ!

* 참고로 해당 이미지는 유튜브 'IT 잡동사니' 님 채널 영상 중 한 컷입니다.
보고 따라하다가 막히는 부분인데 말로만 설명하기가 어려워서 해당 화면 함께 남겨봅니다ㅠ!
![제트브레인 스크립트 작성 시 막히는 부분 (3분 46초)](https://user-images.githubusercontent.com/103179264/164401052-8942694a-f2cb-4e67-bdf3-3361d2aec489.jpg)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 캐릭터 불러오기 코드는 다음 내용을 복사+붙여넣기로 사용해 보세요.

import { ZepetoScriptBehaviour } from 'ZEPETO
.Script'
import { SpawnInfo, ZepetoPlayers, LocalPlayer, ZepetoCharacter } from 
'ZEPETO.Character.Controller'
 
export default class CharacterControllerSample extends ZepetoScriptBehaviour {
   Start() {        
       ZepetoPlayers.instance.CreatePlayerWithZepetoId("", "[ZEPETO_ID]", new SpawnInfo(), 
true);
 
       ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
           let _player : LocalPlayer = ZepetoPlayers.instance.LocalPlayer;
       });
   }
}

공식 가이드 및 샘플 월드를 참고하시면 구현 단계별로 정리가 되어 있습니다. 
https://studio.zepeto.me/kr/guides/character-control
https://github.com/naverz/zepeto-studio-kor/discussions/615

감사합니다.

**Reply by phy9051** [reply]
답변 감사합니다ㅠ 알려주신대로 다시 해봐야겠어요!!
